# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1426195461 14400
#      Thu Mar 12 17:24:21 2015 -0400
# Node ID f0e5c6742716bbeff8188165c4963d12642b72b8
# Parent  6670df97456ca61343235757067660f1fe4b0af7
also request redraw after zooming by a factor (bug #44523)

* graphics.cc (F__zoom__): Also set Vdrawnow_requested after zooming
by a factor.

diff --git a/libinterp/corefcn/graphics.cc b/libinterp/corefcn/graphics.cc
--- a/libinterp/corefcn/graphics.cc
+++ b/libinterp/corefcn/graphics.cc
@@ -11929,12 +11929,13 @@ Undocumented internal function.\n\
     {
       std::string mode = args(1).string_value ();
       double factor = args(2).scalar_value ();
 
       if (error_state)
         return retval;
 
       ax_props.zoom (mode, factor);
+      Vdrawnow_requested = true;
     }
 
   return retval;
 }
